.gitignore

```diff
@@ -4,3 +4,6 @@ coredns.exe
 build/
 release/
 vendor/
+/.idea/coredns.iml
```

Comment:
@neilnaveen Normally don't want to add IDE to gitignore as golang itself is not tied to IDE. Can you remove the change to this file?

Reply:
I have removed all idea folders.